pkg/cache/cache.go

```diff
@@ -156,10 +156,33 @@ type Options struct {
 
 var defaultResyncTime = 10 * time.Hour
 
+// Default defaults the options internals.
+func (options *Options) Default(config *rest.Config) error {
```

Comment:
Why export this?

Reply:
I think this was a leftover, probably can remove